static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -333,14 +339,21 @@ function DefaultGroupEventDetailsContent({
         entryType={EntryType.REQUEST}
         {...eventEntryProps}
       />
-      <div ref={tagsRef}>
-        <EventTagsAndScreenshot event={event} projectSlug={project.slug} />
-      </div>
+      {hasStreamlinedUI ? (
+        <EventTagsDataSection event={event} projectSlug={project.slug} ref={tagsRef} />
+      ) : (
+        <div ref={tagsRef}>
+          <EventTagsAndScreenshot event={event} projectSlug={project.slug} />
+        </div>
+      )}
       <EventContexts group={group} event={event} />
       <EventExtraData event={event} />
       <EventPackageData event={event} />
       <EventDevice event={event} />
       <EventViewHierarchy event={event} project={project} />
+      {hasStreamlinedUI && (
+        <ScreenshotDataSection event={event} projectSlug={project.slug} />
+      )}
```

Comment:
Decided to move screenshot down to the attachments section since it made sense there.